README.md

```diff
@@ -22,6 +22,12 @@
 - Please read _all_ the links included:
 - Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!
 
+### GitHub
+
+- Please create a Github account using your **work** email
```

Comment:
Why do we want to make everyone use their work email for their GH usernames? I believe GH allows for multiple email addresses to be associated with any account that you own. Just want to see what the thought process is here.

Reply:
The thought was to only allow those currently employed to have access. It was mentioned to me to create a GH account with my work email, which meant that I had to edit my git config. As I was going through this process I thought that including our work email to be relevant. If there are no policies in place restricting personal accounts associated with work repositories, then I can remove line 27.

Reply:
GitHub recommends using a single personal account, as mentioned in the links to the answer on this stackexchange answer: https://webapps.stackexchange.com/questions/76618/should-i-use-my-personal-github-account-for-work

I believe creating multiple GH accounts creates a level of personal management complexity which is unnecessary and doesn't bring any obvious benefits, plus it's not a use case encouraged or supported by GH itself. 

My recommendation is that we do not require employees to create a new account. Our GH teams and organizations already offer the necessary isolation and security. Now if someone *wants* to create a different account, I think that is fine. But as a requirement I strongly disagree.

It is still worth clarifying our expectations here since it sounds like contradicting instructions have been given out. Maybe here we can encourage employees to use their personal account and link to this official GH page which also mirrors the suggestions: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts